src/sinks/aws_kinesis/request_builder.rs

```diff
@@ -90,7 +95,7 @@ impl RequestBuilder<KinesisProcessedEvent> for KinesisRequestBuilder {
 
     fn split_input(&self, mut event: KinesisProcessedEvent) -> (Self::Metadata, Self::Events) {
         let metadata = KinesisMetadata {
-            finalizers: event.take_finalizers(),
+            finalizers: event.event.take_finalizers(),
```

Comment:
This is a little gross with the repeated name but c'est la vie

Reply:
Ooo yeah that is a bit gross. I'll take care of that :+1: